unix/src/machine_stats/modules/proc_stats.py

```diff
@@ -115,6 +116,15 @@ def parse_status(process_path):
         stats["name"] = status["name"]
         stats["path"] = "/"
 
+    # Finally, let's see if we can read the username
+    if status.get("uid"):
+        try:
+            uid = int(status["uid"].split()[0])
+            user_entry = getpwuid(uid)
+            stats["user"] = user_entry.pw_name
+        except KeyError:
+            stats["user"] = ""
```

Comment:
I've been thinking why we check for `KeyError` and then discovered, that it is raised by `getpwuid`, so I added the comment to bring more clarity. Also, if we got the UID but failed to get the username for that UID, let's use the UID for username. WDYT?

```suggestion
    # Finally, let's see if we can read the username
    if status.get("uid"):
        uid = int(status["uid"].split()[0])
        try:
            user_entry = getpwuid(
                uid
            )  # KeyError is raised if the entry asked for cannot be found.
        except KeyError:
            stats["user"] = str(uid)  # Fallback to UID
        else:
            stats["user"] = user_entry.pw_name
```

Reply:
Yeah I'm ok with that. 

Sorry, for not making it clear enough _why_ a KeyError was raised 😓, but thanks for the additional comment.